releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java

```diff
@@ -100,9 +91,9 @@ public static Result buildResult(String localRepoPath, String authToken, String
 
         final Result result = new Result();
 
-        final GHRepository remoteRepo = createRemoteRepo(authToken, remoteRepoPath);
+        final GHRepository remoteRepo = CsGitHub.createRemoteRepo(authToken, remoteRepoPath);
         final Set<RevCommit> commitsForRelease =
-            getCommitsBetweenReferences(localRepoPath, startRef, endRef);
+                CsGit.getCommitsBetweenReferences(localRepoPath, startRef, endRef);
```

Comment:
This is an example of where we split out API calls into their own classes so they can easily be mocked.

The easiest way to mock these areas is for them to be `public static`. Special library is needed to mock private methods, and this is the recommended path by the library creators.